web/app/js/index.js

```diff
@@ -72,9 +75,13 @@ class App extends React.Component {
   }
 
   render() {
+    const catalogs = { en: catalogEn, es: catalogEs };
```

Comment:
Good point, I updated it so that we are only passing the necessary catalog into `I18nProvider`.

I tried taking it one step further and dynamically loading the specific catalog we needed into `index.js`:
```
const selectedCatalog = require(`./locales/${selectedLocale}/messages.js`);
```
but I hit the [no-dynamic-require](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-dynamic-require.md) es-lint rule, which seems to be best practices, so I am continuing to load all of the possible catalogs into this file.